sdk/hatchery/hatchery.go

```diff
@@ -16,34 +16,34 @@ import (
 
 // CommonConfiguration is the base configuration for all hatcheries
 type CommonConfiguration struct {
-	Name string
+	Name string `default:"" comment:"Name of Hatchery"`
 	API  struct {
 		HTTP struct {
-			URL      string `default:"http://localhost:8081"`
-			Insecure bool
+			URL      string `default:"http://localhost:8081" comment:"CDS API URL"`
+			Insecure bool   `comment:"sslInsecureSkipVerify, set to true if you use a self-signed SSL on CDS API"`
 		}
 		GRPC struct {
 			URL      string `default:"http://localhost:8082"`
-			Insecure bool
+			Insecure bool   `comment:"sslInsecureSkipVerify, set to true if you use a self-signed SSL on CDS API"`
 		}
-		Token                string `default:"************"`
-		RequestTimeout       int    `default:"10"`
-		MaxHeartbeatFailures int    `default:"10"`
+		Token                string `default:"************" comment:"CDS Token to reach CDS API. See https://ovh.github.io/cds/advanced/advanced.worker.token/ "`
```

Comment:
when we instantiate the default configuration, we have to replace the placeholder by a random key